### DISCUSSION TITLE
Hydrophone/Sonar to record and listen to monsters.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#11832] [Ideas and suggestions]
Basically, this would either be in the same terminal as the sonar or a separate one. With this terminal a crewmember could essentially point the hydrophone and listen to hear various sounds throughout the level at a much greater distance than the sonar reaches. It may require a spawning rework, but ideally one could point the hydrophone forward and check if they hear mudraptors in the distance. 

Another feature might be the ability to record and playback sounds which could create some interesting monster interaction, such as playing mating calls to attract monsters, or playing endworm noises to repel them. 

I think this would also be good for giving the captain more to do than just driving the sub. Additionally it could add to the atmosphere, as maybe you could hear something creepy like husks, or ... it.

**Comment by mygamingaccount** [comment]
https://github.com/Regalis11/Barotrauma/discussions/5219
